das/state.go

```diff
@@ -0,0 +1,54 @@
+package das
+
+import "sync/atomic"
+
+// State is an interface that allows other components to access the
+// current state of the DASer safely.
+type State interface {
```

Comment:
Should `IsRunning()` be added to this interface?

Reply:
The interface was actually unused. I removed it.